CMakeLists.txt

```diff
@@ -22,8 +22,32 @@ include( CTest )
   #set( CMAKE_CXX_VISIBILITY_PRESET hidden )
 
 #---------------------------------------------------------------------
+option( ELASTIX_USE_OPENCL "Use OpenCL enabled GPU" OFF )
+set(_GPU_depends "")
+if(ELASTIX_USE_OPENCL)
+  list(APPEND _GPU_depends ITKGPUCommon)
```

Comment:
More ITKGPUXXX modules might need to go here. Or maybe this is not needed as elastix has its own separate OpenCL-related infrastructure?

Reply:
Yes, elastix has its own internal OpenCL code / classes.